content/en/hardware/concentrators/2n-smartcom-pro.html

```diff
@@ -6,8 +6,8 @@
 
 <br>
 <div class="container">
-	<div class="content dedicated_section">
-	    <img src="/img/hw/LR_2N_SmartCom_Pro_S_Antenama.jpg" style="float: right;" width="450">
+	<div class="content hardware__dedicated-section">
```

Comment:
How about `.hardware__product-promo`? `dedicated-section` says very little about the div.